address_book_ui/src/app/app.routes.ts

```diff
@@ -1,3 +1,16 @@
 import { Routes } from '@angular/router';
+import { RecordComponent } from './record/record.component';
+import { TestComponent } from './test/test.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+    {
+        path: 'test',
+        component: TestComponent,
+        title: "Placeholder 2"
```

Comment:
is the title needed for the nav-bar?

Reply:
I am honestly not sure yet, the title was part of the angular tutorial I was reading to setup some routes